src/Artsy/Analytics/Schema/Interaction.ts

```diff
@@ -34,6 +34,11 @@ export interface Interaction {
    * NOTE: This is old Force schema.
    */
   destination_path?: string
+
+  /*
+  * Flow
+  */
+  flow?: string
```

Comment:
In case you have some context from talking to analytics, mind expand the documentation on this field?

Reply:
I don't actually have context -- @anipetrov what exactly is the "flow" field for?

Reply:
This was moved up from the `AuthenticationInteraction`. I take it to indicate or identify a series of steps (i.e. the authentication flow in the mobile signup view, the order/payment flow for buy now, etc.).

Reply:
Yeah that’s my naive understanding too, I just hoped that maybe there was a nice sentence communicated with @williardx. Let’s just shelf it for now until we have that meeting